build.sbt

```diff
@@ -34,17 +34,19 @@ ThisBuild / wildcardImport := {
 
 // Projects
 
+lazy val projectName: String = "idna4s"
+
 lazy val root = tlCrossRootProject
   .aggregate(
-    bootstring
+    core
   )
-  .settings(name := "idna4s")
+  .settings(name := projectName)
 
-lazy val bootstring = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+lazy val core = crossProject(JVMPlatform, JSPlatform, NativePlatform)
   .crossType(CrossType.Pure)
-  .in(file("bootstring"))
+  .in(file("core"))
   .settings(
-    name := "idna4s-bootstring",
+    name := s"${projectName}-core",
```

Comment:
Do you anticipate any other modules? Or can we just call it `idna4s` and be done :)

Reply:
I'd expect to have at least `idna4s-core-scalacheck` to export `Arbitrary` instances for any newtypes, and that will likely mean we'll have a `idna4s-tests` module so that the tests can use the arbitrary instances, like we do in http4s and elsewhere. 

Otherwise I'm not sure...I suspect it will just be the 1 interesting module, but I'd like to leave the door open on that until we are a bit further along.